.eslintrc.js

```diff
@@ -1,28 +1,17 @@
-// http://eslint.org/docs/user-guide/configuring
-
 module.exports = {
   root: true,
-  parser: 'babel-eslint',
-  parserOptions: {
-    sourceType: 'module'
-  },
   env: {
-    browser: true,
+    node: true,
   },
-  // https://github.com/standard/standard/blob/master/docs/RULES-en.md
-  extends: 'standard',
-  // required to lint *.vue files
-  plugins: [
-    'html'
+  extends: [
+    // consider switching to `plugin:vue/strongly-recommended` or `plugin:vue/recommended` for stricter rules.
+    'plugin:vue/essential',
```

Comment:
I saw that my template vue-cli application uses the same setting and it also seems to comply to our existing rule set. If you ask me it`s fine to stick with it for now and possibly worth changing it in a separate run?!

Reply:
That comment was present inside the last vue-cli2 template (dbc9e8ba635d3597eef0f3e66b811cee60cbed33) and was removed in vue-cli3. However, I've let it here as a remainer in case you wanted to harden your code style afterwards.